src/Symfony/EventListener/ReadListener.php

```diff
@@ -142,6 +144,8 @@ public function onKernelRequest(RequestEvent $event): void
                 || ($operation instanceof Put && !($operation->getAllowCreate() ?? false))
             )
         ) {
+            //Log it in debug in case the dev just forgot to implement it
+            $this->logger->debug("No provider found for " . $resourceClass);
             throw new NotFoundHttpException('Not Found');
```

Comment:
Maybe it could be useful to also pass the previous exception here 🙏

Reply:
I could add the exception message or even the stacktrace, but that seems a bit overkill doesn't it? Having a log saying something like this should be fine, shouldn't it?

`app.DEBUG: No provider registered for App\ApiResource\TestResource {"resource_class":"App\\ApiResource\\TestResource"} []`

Reply:
sorry, misunderstood what you meant. I'll do that.

Reply:
@mrossard , thank you! 🙏